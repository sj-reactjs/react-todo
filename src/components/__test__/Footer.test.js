import { render, screen } from '@testing-library/react'
import Footer from './../Footer'

describe(`Footer component`, () => {
    it(`render`, () => {
        const { asFragment } = render(<Footer />)
        expect(asFragment()).toMatchSnapshot()
    })
})