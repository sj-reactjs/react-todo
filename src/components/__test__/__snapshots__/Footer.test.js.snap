// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component render 1`] = `
<DocumentFragment>
  <footer>
    Footer !!
  </footer>
</DocumentFragment>
`;
